jasminewd/index.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for jasminewd2 0.1.1
```

Comment:
Moving the folder to jasminewd2. The published npm is "jasminewd2" however, the project is called "jasminewd"